src/library_pipefs.js

```diff
@@ -0,0 +1,208 @@
+mergeInto(LibraryManager.library, {
+  $PIPEFS__postset: '__ATINIT__.push(function() { PIPEFS.root = FS.mount(PIPEFS, {}, null); });',
```

Comment:
I think, it may be worth looking at sockfs -- it seems to have the same dependency.